setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
```

Comment:
When I try to execute `qccalculator --help` or just `qccalculator` I get `ImportError: module 'qccalculator.cli' has no attribute 'start'`. You probably also need to adjust the 'console_scripts' entry point.

Reply:
This works in my environment pretty well:
```
    entry_points={
        'console_scripts': ['qccalculator = qccalculator.cli:cli']
      }
```
After that you can directly install the package via pip and the console scripts will be available automatically.

Reply:
I just had to rebuild the container after the changes to the setup.py (where it pip installs the qccalculator package) and then I could do `qccalculator full -h` directly for example.

Reply:
solve in the last commit